libpod/container_internal_linux.go

```diff
@@ -482,6 +482,19 @@ func (c *Container) checkpoint(ctx context.Context, options ContainerCheckpointO
 	if c.state.State != ContainerStateRunning {
 		return errors.Wrapf(ErrCtrStateInvalid, "%q is not running, cannot checkpoint", c.state.State)
 	}
+
+	// Create the CRIU log file and label it
+	dumpLog := filepath.Join(c.bundlePath(), "dump.log")
+
+	logFile, err := os.OpenFile(dumpLog, os.O_RDWR|os.O_CREATE, 0600)
+	if err != nil && os.IsNotExist(err) {
+		return errors.Wrapf(err, "failed to create CRIU log file %q", dumpLog)
+	}
+	logFile.Close()
```

Comment:
`logFile` could be `nil` here, as we are testing only for the `os.IsNotExist(err)` case

Reply:
Thanks, I think i was mainly confused by the flags to create to file. I removed it and changed the flags in OpenFile() to correctly create the file.